src/porting.h

```diff
@@ -360,6 +360,9 @@ inline u32 getDeltaMs(u32 old_time_ms, u32 new_time_ms)
 	}
 #elif defined(_WIN32)
 	inline void setThreadName(const char* name) {}
+#elif defined(__GNU__)
+//#warning "GNU/Hurd platform, pthread_setname_np not yet supported"
+inline void setThreadName(const char* name) {}
```

Comment:
On Tue, 20. Jan 10:14 ShadowNinja <notifications@github.com> wrote:
[..]
>    You can actually combine this with the Windows section too.

GNU/Hurd is not Windows. A change in WIN32 or the default might have a
negative impact on GNU/Hurd again in the future. Hence I think it is
reasonable to make this distinction clear.